src/Miningcore/Payments/PaymentSchemes/PPSPaymentScheme.cs

```diff
@@ -251,10 +334,10 @@ private void LogDiscardedShares(PoolConfig poolConfig, Block block, DateTime val
             }
 
             // this should never happen
-            if (blockRewardRemaining <= 0 && !done)
```

Comment:
Why is it removed? did we anticipate negative rewards in calc?

Reply:
Since we are not dealing with remaining Block rewards. Block reward is calculated based on hashrate/block frequency/payout time etc.